source/common/config/well_known_names.cc

```diff
@@ -132,6 +134,14 @@ TagNameValues::TagNameValues() {
   // internal state of the regex which performs better.
   addRe2(HTTP_CONN_MANAGER_PREFIX, R"(^listener\..*?\.http\.((<TAG_VALUE>)\.))", ".http.");
 
+  // Extract ext_authz stat_prefix field
+  if (Runtime::runtimeFeatureEnabled("envoy.restart_features.ext_authz_prefix_handling")) {
```

Comment:
Unless something has changed since last time I investigated this, the runtime guard will not work because this is resolved early during startup before the runtime is initialized/loaded/configured. For past changes, we've just documented the change and not added a runtime guard at all, because there wasn't a good way to add it.

Reply:
Got it - I'll rip all the runtime guard bits out then.